MVNDegenerate test fails with TFP 0.20.1

Here's the error message:

```
FAILED tests/distributions/test_mvn_degen.py::TestMVNDegenerateAgainstTensorFlow::test_multiple_samples - ValueError: Incompatible shapes for broadcasting: (1, 3, 5, 1) and requested shape (2, 2, 5, 1)
```

See also our [CI log](https://github.com/liesel-devs/liesel/actions/runs/5221892365/jobs/9426719770). Downgrading to TFP 0.19 solves the issue, but we should probably investigate more thoroughly what's going on...

--- a/tests/distributions/test_mvn_degen.py
+++ b/tests/distributions/test_mvn_degen.py
@@ -826,11 +826,13 @@ class TestMVNDegenerateAgainstTensorFlow:
             mvnd.log_prob(sample)
 
         # The tfd version apparently does not always raise the same error if shapes
-        # are not compatible for broadcasting. Here, it raises a TypeError.
+        # are not compatible for broadcasting.
+        # Here, it raises a TypeError for TFP v0.19 and smaller and a ValueError from
+        # TFP v0.20.1.
         # Sample shape: (1, 3, 5)
         # Batch shape: (2, 2)
         sample = jnp.array([[beta, beta, beta]])
-        with pytest.raises(TypeError):
+        with pytest.raises((TypeError, ValueError)):
             mvn.log_prob(sample)
 
         with pytest.raises(ValueError):

